pkgs/tools/networking/p2p/tahoe-lafs/default.nix

```diff
@@ -56,7 +56,7 @@ pythonPackages.buildPythonApplication rec {
   propagatedBuildInputs = with pythonPackages; [
     twisted foolscap nevow simplejson zfec pycryptopp darcsver
     setuptoolsTrial setuptoolsDarcs pycrypto pyasn1 zope_interface
-    service-identity pyyaml
+    service-identity pyyaml magic-wormhole hypothesis treq
```

Comment:
I don't think that Hypothesis needs to be propagated; could you instead put it in `checkInputs` or possibly `buildInputs`?

Reply:
@MostAwesomeDude I added it to checkInputs, and everything seems to still work